### DISCUSSION TITLE
Can't change source and translation font size in Coursera.org video

### DISCUSSION THREAD

**Discussion by nielpattin** [immersive-translate/immersive-translate#2041] [Help] [answered]
### 概述 | Summary

First, thank you for the extension.
Second, like what the title says. I also tried to "edit full user config" in the developer settings tab for these two values but I can't go below 50%.
Any idea?
```
"sourceFontSize": "30",
"translationFontSize": "30",
```

**Comment by theowenyoung** [comment, marked as answer]
This is expected, the site's subtitles are controlled by the original site and cannot be changed
